xapian-letor/ranker/ranker.cc

```diff
@@ -239,10 +239,11 @@ initialise_queryparser(const Xapian::Database & db)
     Xapian::QueryParser parser;
     parser.add_prefix("title", "S");
     parser.add_prefix("subject", "S");
+    parser.add_prefix("description", "XD");
     parser.set_database(db);
     parser.set_default_op(Xapian::Query::OP_OR);
     parser.set_stemmer(stemmer);
-    parser.set_stemming_strategy(Xapian::QueryParser::STEM_SOME);
+    parser.set_stemming_strategy(Xapian::QueryParser::STEM_ALL_Z);
```

Comment:
What's the reason for changing the stemming strategy here?

(Indeed there's no useful explanation for either of the two changes that are effectively being made in this commit in the commit message.)

Reply:
fixed in a fixup commit.